src/main/scala/xiangshan/mem/vector/VSegmentUnit.scala

```diff
@@ -356,7 +359,9 @@ class VSegmentUnit (implicit p: Parameters) extends VLSUModule
     instMicroOp.exceptionVl.valid     := false.B
     instMicroOp.exceptionVl.bits      := io.in.bits.src_vl.asTypeOf(VConfig()).vl
     segmentOffset                     := 0.U
-    instMicroOp.isFof                 := (fuOpType === VlduType.vleff) && FuType.isVLoad(fuType)
+    instMicroOp.isFof                 := (fuOpType === VlduType.vleff) && isVSegLoad //FuType.isVLoad(fuType)
```

Comment:
This does not seem right.

Reply:
The `instMicroOp.isVSegLoad` is not set until the next cycle, so `instMicroOp.isFof := (fuOpType === VlduType.vleff) && isVSegLoad` seems to be wrong.
Should use `instMicroOp.isFof := (fuOpType === VlduType.vleff) && FuType.isVSegLoad(io.in.bits.uop.fuType)`.

Reply:
Okay, I'll fix it later

Reply:
Blame @weidingliu XD